src/components/LayerControl.vue

```diff
@@ -91,7 +91,6 @@ export default {
 
 .layer-control {
   background-color: var(--white);
```

Comment:
what css framework are we using?  Sass? Postcss? Last time we used postcss it was properly expanding the variables and this caused some problems.